src/styles/main.scss

```
@@ -0,0 +1,21 @@
+@import "/styles/utils/variables.scss";
+@import "/styles/blocks/stopWatch.scss";
+
+body {
+  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+  background-size: 200% 200%;
```

Just a fix on background 😊
```suggestion
  background-size: 300% 300%;
  background-repeat: no-repeat;
```

---

Btw, you can move all of this general codes (the ones you're applying on html, body, ...) into:
`utils -> _extends.scss`
and then import it in here...